.ci/travis/openjdk14-test/jdk14-test-files.list

```diff
@@ -0,0 +1,2548 @@
+test/langtools/tools/jdeps/modules/upgrademodulepath/UpgradeModulePathTest.java
+test/langtools/tools/jdeps/unsupported/JDKUnsupportedTest.java
+test/langtools/tools/sjavac/CompileCircularSources.java
+test/langtools/tools/sjavac/CompileExcludingDependency.java
+test/langtools/tools/sjavac/CompileWithAtFile.java
+test/langtools/tools/sjavac/CompileWithInvisibleSources.java
+test/langtools/tools/sjavac/CompileWithOverrideSources.java
+test/langtools/tools/sjavac/HiddenFiles.java
+test/langtools/tools/sjavac/IdleShutdown.java
+test/langtools/tools/sjavac/IncCompInheritance.java
+test/langtools/tools/sjavac/IncCompileChangeNative.java
+test/langtools/tools/sjavac/IncCompileDropClasses.java
+test/langtools/tools/sjavac/IncCompileNoChanges.java
+test/langtools/tools/sjavac/IncCompileUpdateNative.java
+test/langtools/tools/sjavac/IncludeExcludePatterns.java
+test/langtools/tools/sjavac/JavacOptionPrep.java
+test/langtools/tools/sjavac/NoState.java
+test/langtools/tools/sjavac/OptionDecoding.java
+test/langtools/tools/sjavac/OverlappingSrcDst.java
+test/langtools/tools/sjavac/PackagePathMismatch.java
+test/langtools/tools/sjavac/ParallelCompilations.java
+test/langtools/tools/sjavac/PermittedArtifact.java
+test/langtools/tools/sjavac/PooledExecution.java
+test/langtools/tools/sjavac/Serialization.java
+test/langtools/tools/sjavac/StateDir.java
+##----------------Checkstyle cannot parse below files----------------##
+#test/langtools/tools/javac/unicode/UnicodeCommentDelimiter.java
+#test/langtools/tools/javac/modules/T8168854/module-info.java
+#test/langtools/tools/doclint/moduleTests/good/module-info.java
+#test/langtools/tools/doclint/packageTests/good/package-info.java
+#test/langtools/tools/javac/processing/model/util/printing/module-info.java
+#test/langtools/tools/javac/IllegalAnnotation.java
+#test/langtools/tools/javac/MethodParameters/UncommonParamNames.java
+#test/langtools/tools/javac/TextBlockLang.java
+#test/langtools/tools/javac/annotations/typeAnnotations/8013180/QualifiedName.java
+#test/langtools/tools/javac/annotations/typeAnnotations/failures/CheckErrorsForSource7.java
+#test/langtools/tools/javac/annotations/typeAnnotations/failures/T8011722.java
+#test/langtools/tools/javac/annotations/typeAnnotations/newlocations/AllLocations.java
+#test/langtools/tools/javac/annotations/typeAnnotations/newlocations/NestedTypes.java
+#test/langtools/tools/javac/enum/LocalEnum.java
+#test/langtools/tools/javac/annotations/pos/TrailingComma.java
+#test/langtools/tools/javac/enum/TrailingComma.java
+#test/langtools/tools/javac/parser/SingleCommaAnnotationValue.java
+#test/langtools/tools/javac/treeannotests/AnnoTreeTests.java
+#test/langtools/tools/javac/unicode/SupplementaryJavaID1.java
+#test/langtools/tools/javac/unicode/SupplementaryJavaID6.java
+#test/langtools/tools/javac/unicode/UnicodeAtEOL.java
+#test/langtools/tools/javac/unicode/UnicodeCommentDelimiter.java
+##-----------------Add new files below this line---------------------##
```

Comment:
This preserves the list found on the openjdk website.  Above this line is the entire list.  Below, we can add new files that we discover.